### PR TITLE
feature: improve static texts

### PR DESCRIPTION
**Pull request by herrardo**
EX-64762

Change texts to improve and complete identifier results in spanish

# Pull request template
 
Describe the purpose of the change, the specific changes done in detail, and the issue you have fixed.

## Motivation and context

- [ ] Dependencies. If any, specify:
- [x] Open issue. If applicable, link: EX-6476

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that causes existing functionality to not work as expected)
- [ ] Change requires a documentation update

## What is the destination branch of this PR?

- [x] `Main`
- [ ] Other. Specify:

## How has this been tested?

Tests performed according to [testing guidelines](https://github.com/empathyco/x/blob/main/.github/contributing/tests.md): 

## Checklist:

- [ ] My code follows the **[style guidelines](https://github.com/empathyco/x/blob/main/.github/CONTRIBUTING.md#style-guides)** of this project.
- [ ] I have performed a **self-review** on my own code.
- [ ] I have **commented** my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate **no new warnings**.
- [ ] I have added **tests** that prove my fix is effective or that my feature works.
- [ ] New and existing **unit tests pass locally** with my changes.

### CODE REVIEW

File: src/i18n/messages/es.messages.json [hunk 1]
```diff
@@ -13,11 +13,14 @@
...
-      "title": "Para ti"
+      "title": "Quizás te interese"
     },
     "nextQueriesGroup": {
-      "title": "Te podría interesar",
-      "message": "Esto es lo que otros compradores han buscado después de <span class=\"x-text x-text--bold\">\"{query}\"</span>"
+      "title": "Quizás te interese",
+      "message": "Los que buscaron <span class=\"x-text x-text--bold\">\"{query}\"</span> también han visto:"
+    },
+    "identifierResults": {
```
Comment (tiborux) [inline src/i18n/messages/es.messages.json:22]: Remove the idenftifierResults translation

File: src/i18n/messages/en.messages.json [hunk 2]
```diff
@@ -13,14 +13,14 @@
...
     "nextQueries": {
-      "title": "Next Queries"
+      "title": "What's Next"
     },
     "nextQueriesGroup": {
       "title": "You may be interested",
-      "message": "This is what other shoppers searched after <span class=\"x-text x-text--bold\">\"{query}\"</span>"
+      "message": "Those who searched for <span class=\"x-text x-text--bold\">\"{query}\"</span> also viewed:"
     },
     "identifierResults": {
```
Comment (tiborux) [inline src/i18n/messages/en.messages.json:22]: Remove the idenftifierResults translation

Comment (herrardo) [reply src/i18n/messages/en.messages.json]: done

File: src/i18n/messages/es.messages.json (hunk 1 above)
Comment (herrardo) [reply src/i18n/messages/es.messages.json]: done

**Review by tiborux — APPROVED** [review]

**Review by javieri-empathy — APPROVED** [review]

**Pull request merged**